pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -22,12 +22,11 @@ let
       src = fetchgit {
         url = "https://review.coreboot.org/coreboot";
         rev = version;
-        sha256 = "073n8yid3v0l9wgwnrdqrlgzaj9mnhs33a007dgr7xq3z0iw3i52";
+        sha256 = "sha256-PCum+IvJ136eZQLovUi9u4xTLLs17MkMP5Oc0/2mMY4=";
         fetchSubmodules = false;
         leaveDotGit = true;
         postFetch = ''
-          patchShebangs $out/util/crossgcc/buildgcc
```

Comment:
The patching is done since we run the the script below, as you can see. So to make sure that the script works correctly, I want to leave this.

Reply:
Addressed [here](https://github.com/NixOS/nixpkgs/pull/171223#pullrequestreview-958985608).